blivetgui/communication/server.py

```
@@ -207,17 +202,27 @@ def _pickle_answer(self, answer):
 
         return pickled_answer
 
+    def _get_proxy_object(self, proxy_id):
+        """ Look up a proxy object by its ProxyID, raising KeyError with
+            a descriptive message if not found.
+        """
+        try:
+            return self.object_dict[proxy_id.id]
+        except KeyError:
+            raise KeyError("Unknown proxy object with id %s" % proxy_id.id)
```

Ruff (0.15.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)